### DISCUSSION TITLE
While converting PNG with transparency to CMYK-TIF, transparency is discarded

### DISCUSSION THREAD

**Discussion by fuzzy-andreas** [ImageMagick/ImageMagick#7584] [Help] [answered]
### ImageMagick version

7.1.1-Q16-HDRI

### Operating system, version and so on

Windows Server 2016 Std/CLI

### Description

If I convert a PNG with transparency to an RGB TIF using IM, the transparency remains.
If I convert the PNG to a CMYK TIF, the transparency is missing.
Is there a way to preserve the transparency in the CMYK TIF using a CLI command?

I have attached an example file.
![RGB_transparency](https://github.com/user-attachments/assets/c51a070b-9afb-43d6-a8d3-5f72532c0b7b)

Regards,
Andreas

**Comment by snibgo** [comment, marked as answer]
It works fine for me, using IM v7.1.1-20 on Windows 11.

IM has two methods for converting from sRGB to CMYK, either `-colorspace` or `-profile`:

```
magick giraffe.png -colorspace CMYK t_cmyk.tiff

magick giraffe.png -profile sRGB.icc -profile USWebCoatedSWOP.icc t_cmyk2.tiff
```
Both tiff results have CMYK and alpha channels.

Please show the command you used, and why you think the tiff has no transparency.

**Reply by fuzzy-andreas** [reply]
Hi Snibgo, thank you for the help! The problem was not IM, but in the environment. I control the CLI from Enfocus Switch and something didn't work here.
